src/Symfony/Component/WebLink/composer.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "symfony/weblink",
```

Comment:
The (new) problem with Link is that it's not an implementation of the PSR-13, it just use it. It maybe confusing :)

`symfony/link` and `symfony/web-link` are fine to me. It's time to start a Twitter poll?